src/utils/constants.ts

```diff
@@ -1,16 +1,28 @@
-const mainCriteriaNames: { [s: string]: string } = {
-  reliability: 'Reliable & not misleading',
-  pedagogy: 'Clear & pedagogical',
-  importance: 'Important and actionable',
-  layman_friendly: 'Layman-friendly',
-  entertaining_relaxing: 'Entertaining and relaxing',
-  engaging: 'Engaging & thought-provoking',
-  diversity_inclusion: 'Diversity & inclusion',
-  better_habits: 'Encourages better habits',
-  backfire_risk: 'Resilience to backfiring risks',
-};
+export const mainCriteriaNames: string[][] = [
+  ['reliability', 'Reliable & not misleading'],
+  ['pedagogy', 'Clear & pedagogical'],
+  ['importance', 'Important and actionable'],
+  ['layman_friendly', 'Layman-friendly'],
+  ['entertaining_relaxing', 'Entertaining and relaxing'],
+  ['engaging', 'Engaging & thought-provoking'],
+  ['diversity_inclusion', 'Diversity & inclusion'],
+  ['better_habits', 'Encourages better habits'],
+  ['backfire_risk', 'Resilience to backfiring risks'],
+];
 
-const allCriteriaNames = { ...mainCriteriaNames };
-allCriteriaNames['largely_recommended'] = 'Should be largely recommended';
+export const mainCriteriaNamesObj: { [c: string]: string } =
+  mainCriteriaNames.reduce((a: { [c: string]: string }, [k, v]) => {
+    a[k] = v;
+    return a;
+  }, {});
```

Comment:
It could be simplified with [`Object.fromEntries`](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries):

```suggestion
export const mainCriteriaNamesObj: { [c: string]: string } =  Object.fromEntries(mainCriteriaNames);
```

Reply:
Nice